.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: Test
+
+on: [pull_request]
```

Comment:
I think we want to use `[push, pull_request]` or else tests won't run for pushes created after a PR is created and when a PR is merged into main branch

Reply:
Thanks, I turned this off when I was testing to reduce noise 👍 

I'll do the same all all other PRs.